pkg/controllers/updaterun/execution_integration_test.go

```diff
@@ -972,6 +970,349 @@ var _ = Describe("UpdateRun execution tests", func() {
 	})
 })
 
+var _ = Describe("UpdateRun execution tests - delete ClusterApprovalRequest, don't recreate", func() {
```

Comment:
Thank you for writing this test! We should definitely figure out a way to make this test more extensible. Right now it's too difficult to modify.